tests/oadp/datamover/test_vsr.py

```diff
@@ -0,0 +1,37 @@
+import os
+from pathlib import Path
+
+import pytest as pytest
+
+from oadp_resources.oadp.datamover.volume_snapshot_restore import VolumeSnapshotRestore
+
+path = str(Path(__file__).parent.parent.parent)
```

Comment:
what is this please write some comment ehat to be expacted or use some other name ROOT_DIR ? PATH_TO_TEST_SAMPLS?

Reply:
done